addons/website_sale/static/src/js/website_sale.js

```diff
@@ -169,11 +169,31 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
     start: function () {
         var def = this._super.apply(this, arguments);
 
+        var hash = window.location.hash.substring(1);
+        if (hash) {
+            var decodedHash = decodeURIComponent(hash);
+            var idx = decodedHash.search(/variants=\d/g);
```

Comment:
`params = $.deparam(hash)` => `params['variants']` -> `'x,y,z'`
(no need of decode, search, substring, split, split, ...)